### PR TITLE
netlink: reduce impact of NetNsName resolution

### PR DESCRIPTION
**Pull request by safchain**

### CODE REVIEW

**Comment by safchain** [comment]
run skydive-functional-tests-backend-elasticsearch

**Comment by safchain** [comment]
run skydive-functional-tests-backend-orientdb

**Comment by safchain** [comment]
run skydive-functional-tests-backend-elasticsearch

**Comment by safchain** [comment]
rerun skydive-cdd-overview-tests

**Comment by safchain** [comment]
run skydive-functional-tests-backend-elasticsearch

File: topology/probes/netlink/netlink.go [hunk 1]
```diff
@@ -360,7 +361,7 @@ func newInterfaceMetricsFromNetlink(link netlink.Link) *topology.InterfaceMetric
 	}
 }
 
-func (u *NetNsProbe) updateLinkNetNsName(intf *graph.Node, link netlink.Link, metadata graph.Metadata) {
+func (u *NetNsProbe) updateLinkNetNsName(intf *graph.Node, link netlink.Link, metadata graph.Metadata) bool {
```
Comment (masco-enovance) [inline topology/probes/netlink/netlink.go]: look like we are not using the return value, do we really need this change?

Comment (safchain) [reply topology/probes/netlink/netlink.go]: right I forgot something :)

**Comment by safchain** [comment]
rerun skydive-functional-tests-backend-elasticsearch

**Pull request merged**